### PR TITLE
feat: github button on about and discord icon on buttons in about

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] (if relevant) I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] (if relevant) I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3434** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3434" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3434?utm_source=stack-comment).

File: src/components/AboutDashboard/components/AboutTheProtocolSection.tsx [hunk 1]
```diff
@@ -27,6 +30,13 @@ export const AboutTheProtocolSection = () => {
             ),
           )}
         </div>
+
+        <ExternalLink href="https://github.com/jbx-protocol/juice-interface">
+          <AboutButton className="flex items-center gap-2">
```
Comment (tomquirk) [inline src/components/AboutDashboard/components/AboutTheProtocolSection.tsx:35]: @wraeth-eth i think we should use antd button for the meantime, for styling consistency (slight differences in height, hover effect, dark/light mode etc.)

Comment (tomquirk) [reply src/components/AboutDashboard/components/AboutTheProtocolSection.tsx:35]: Heads up: i switched over the existing aboutbuttons to antd buttons 🙏

Comment (wraeth-eth) [reply src/components/AboutDashboard/components/AboutTheProtocolSection.tsx]: Thanks! Updated

**Pull request merged**